pallets/identity-management/src/lib.rs

```diff
@@ -102,6 +103,19 @@ pub mod pallet {
 		DelegateeNotExist,
 		/// a `create_identity` request from unauthorised user
 		UnauthorisedUser,
+
+		/// copy from litentry_primitives::IMPError
```

Comment:
or can we simply use these errors in tee-worker?

Reply:
I still haven't found how to use these errors in tee-worke or dispatch custom error(from extrinsic paramater) in parachain.:frowning_face: